src/extensions/markdown/CodeBlock/CodeBlockHighlight/CodeBlockHighlight.ts

```diff
@@ -11,11 +11,12 @@ import {Step} from 'prosemirror-transform';
 import {findChildrenByType} from 'prosemirror-utils';
 import {Decoration, DecorationSet} from 'prosemirror-view';
 
-import {ExtensionAuto} from '../../../core';
-import {codeBlockLangAttr, codeBlockNodeName, codeBlockType} from '../../../extensions/markdown';
-import {capitalize} from '../../../lodash';
-import {logger} from '../../../logger';
-import {codeLangSelectTooltipViewCreator} from '../CodeMirrorView/TooltipPlugin';
+import {ExtensionAuto} from '../../../../core';
+import {capitalize} from '../../../../lodash';
+import {logger} from '../../../../logger';
+// TODO: check cycle imports
+import {codeLangSelectTooltipViewCreator} from '../../../behavior/CodeMirrorView/TooltipPlugin';
```

Comment:
We need to move or copy TooltipPlugin from CodeMirrorView to CodeBlockHighlight extension, because CodeMirrorView extension well be removed soon